### DISCUSSION TITLE
when will http2 tls version be out? also rpc

### DISCUSSION THREAD

**Discussion by ouvaa** [bytedance/monoio#249] [Q&A]
1. when will http2 tls version be out?
2. when will rpc be out?
3. not combining with volo?

sorry curious about ETA. possible to enlighten? been looking at the repo for quite some time.
it's impressive progress.

**Comment by ihciah** [comment]
Native h2 with tls is supported now, also you can use hyper.
There’s no native rpc plan within 2 months. You can use other rpc framework with pollio feature.
